pkg/kanctl/actionset.go

```diff
@@ -121,7 +124,7 @@ func initializeAndPerform(cmd *cobra.Command, args []string) error {
 	ctx := context.Background()
 	valFlag, _ := cmd.Flags().GetBool(skipValidationFlag)
 	if !valFlag {
-		err = verifyParams(ctx, params, cli, crCli, osCli)
+		err = verifyParams(cmd, ctx, params, cli, crCli, osCli)
```

Comment:
instead of passing whole `cmd` object just to get value to `wait` flag can we parse it here and just pass the boolean parameter?

Reply:
Done !